vue-components/package.json

```
@@ -19,6 +19,7 @@
     "test": "npm-run-all test:*",
     "test:lint": "vue-cli-service lint --no-fix",
     "test:lintcss": "stylelint '**/*.vue'",
+    "test-unit-coverage": "vue-cli-service test:unit -c ./jest.config.js --coverage && sed -i 's|SF:/src/client/data-bridge/|SF:|g' coverage/lcov.info",
```

The data-bridge part probably shouldn't be there and it doesn't look like it's needed.

```suggestion
    "test-unit-coverage": "vue-cli-service test:unit -c ./jest.config.js --coverage",
```

---

erm, this didn't change :eyes:

---

The rebase erased the suggested change I accepted 😬